src/LVMGenerator.h

```diff
@@ -409,17 +415,11 @@ class LVMGenerator : protected RamVisitor<void, size_t> {
         code->push_back(symbolTable.lookup(aggregate.getRelation().getName()));
         code->push_back(symbolTable.lookup(types));
 
-        if (aggregate.getFunction() == RamAggregate::COUNT) {  // To count, there is no need to iterate
+        // Special case for counting
```

Comment:
I approve of the change to non-inline comments, but it's now duplicating the line it comments so can be deleted.